app/src/app/pages/servers/servers.component.html

```diff
@@ -9,6 +9,8 @@
         </button>
         <div *ngIf="adding">
           <input type="text" placeholder="user:password@hostname:port" [(ngModel)]="newServer" [disabled]="loading" />
+          <input type="text" placeholder="URL to retrieve server types" [(ngModel)]="newTypesUrl"
+            [disabled]="loading" />
```

Comment:
maybe this could be controlled by an env var?

Reply:
it's not mandatory, but yes why not

Reply:
(im wondering if some other people are using Mongoku in prod btw)

Reply:
it can be some "experimental" feature flag maybe

Reply:
mmm not sure it's worth the hassle for one feature, this is old technology we're working with here - no simple way to pass the value of a server-side env var to the front